docs/spec/v2beta1/helmreleases.md

```diff
@@ -678,6 +678,10 @@ The `spec.chart.spec.sourceRef` is a reference to an object managed by
 [revision](https://github.com/fluxcd/source-controller/blob/main/docs/spec/v1beta1/common.md#source-status)
 changes, it generates a Kubernetes event that triggers a new release.
 
+> **Note** that on multi-tenant clusters,  platform admins can disable cross-namespace references
```

Comment:
Please move this below after "The `HelmChart` is created in the same namespace as the `sourceRef`".